src/components/post-card/PostCard.js

```diff
@@ -113,9 +116,11 @@ class PostCard extends PureComponent {
         }
         { !!error &&  <div>{error}</div> }
 
-        {/* todo 3 : добавить ссылку Link на урлу с айди поста, где будут детали поста
+        {/* donetodo 3 : добавить ссылку Link на урлу с айди поста, где будут детали поста
                     по аналогии с 24 строкой в UserCard
         */}
+        {(url === "/posts") &&
```

Comment:
Good
Also, you can check if there is id in props.match
in case  if the component is rendered as page there would be id
otherwise (for example in list) there wouldn't be id
and we need to render this link only in case if the component is rendered not like page